### PR TITLE
my homework 2 and 3

### PR DESCRIPTION
**Pull request by andbelvg**

### CODE REVIEW

**Review by pluhin — CHANGES_REQUESTED** [review]
Please use correct github markdown

**Comment by pluhin** [comment]
Any updates?

**Comment by andbelvg** [comment]
Markdown updated

File: Andrei_Dubina/3 - Git remote/3 - Git remote.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Created local repository
+
+## Generated SSH keys locally 
+by using ssh-keygen.
+
+Checked the SSH key files, added them in my github account in settings. 
+
+## Connected to remote repository: 
+git remote add origin git@github.com:andbelvg/03-git-homework.git
```
Comment (pluhin) [inline Andrei_Dubina/3 - Git remote/3 - Git remote.md:9]: Please use code section for code

File: Andrei_Dubina/3 - Git remote/3 - Git remote.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Pushed local repository to remote 
+by using  git push -u origin --all
```
Comment (pluhin) [inline Andrei_Dubina/3 - Git remote/3 - Git remote.md:12]: Please use code section for code

**Comment by pluhin** [comment]
Please squash all your commits

**Comment by pluhin** [comment]
I do not see links to your external repositories for 3th task

File: Andrei_Dubina/3 - Git remote/3 - Git remote.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Connected to Slack: 
+Subscribed to **all commits**
+
+Received the first notification about commit: 
+andbelvgandbelvg
+1 new commit pushed to master
+206a2c35 - hey hey hey
+<https://github.com/andbelvg/03-git-homework|andbelvg/03-git-homework>andbelvg/03-git-homework
```
Comment (pluhin) [inline Andrei_Dubina/3 - Git remote/3 - Git remote.md:21]: What is it. It is broken, I mean url

**Review by pluhin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**